test/unit/math/prim/mat/fun/resize_test.cpp

```diff
@@ -10,12 +10,12 @@ using stan::math::row_vector_d;
 
 TEST(MathMatrix, resize_double) {
```

Comment:
Again, just changing resize to work with ints and match dims with the VectorXd and RowVectorXd size semantics.